README.md

```diff
@@ -1,2 +1,79 @@
-# Hello-Microverse-Review
-Setup of Microverse hello project for TSE training
+![](https://img.shields.io/badge/Hello-Microverse-blue)
+
+# Hello Microverse Project
+
+> Hello Microverse Project is simple app built  during microverse bootcamp which helps one to get familiar with setting up basic project on github
+
+![screenshot]()
```

Comment:
- [ ]  I can see that your screen shoot photo is empty. To make your README file more professional, I would recommend you add your project image. 
![image](https://user-images.githubusercontent.com/43877689/169067440-cbb61309-5cd8-4b50-afc3-91de621413d2.png)